docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -0,0 +1,213 @@
+
+[.hljs-theme-light.nopadding]
+```javascript
+use openzeppelin::account::Account;
+```
+Base Account contract implementation.
+
+[.contract-index]
+.Utilities
+--
+* xref:#Account-assert_only_self[`++InternalImpl::assert_only_self(self)++`]
+--
+
+[.contract-index]
+.Functions
+--
+* xref:#Account-\\__validate_deploy__[`++__validate_deploy__(self, hash, signature)++`]
+
+[.contract-subindex-inherited]
+.SRC6Impl
+
+* xref:#Account-\\__execute__[`++__execute__(self, calls)++`]
+* xref:#Account-\\__validate__[`++__validate__(self, calls)++`]
+* xref:#Account-is_valid_signature[`++is_valid_signature(self, hash, signature)++`]
+
+[.contract-subindex-inherited]
+.SRC5Impl
+
+* xref:#Account-supports_interface[`++supports_interface(self, interface_id)++`]
+
+[.contract-subindex-inherited]
+.DeclarerImpl
+
+* xref:#Account-\\__validate_declare__[`++__validate_declare__(self, class_hash)++`]
+
+[.contract-subindex-inherited]
+.PublicKeyImpl
+
+* xref:#Account-set_public_key[`++set_public_key(self, new_public_key)++`]
+* xref:#Account-get_public_key[`++get_public_key(self)++`]
+
+[.contract-subindex-inherited]
+.InternalImpl
+
+* xref:#Account-initializer[`++initializer(self, _public_key)++`]
+* xref:#Account-validate_transaction[`++validate_transaction(self)++`]
+* xref:#Account-_set_public_key[`++_set_public_key(self, new_public_key)++`]
+* xref:#Account-_is_valid_signature[`++_is_valid_signature(self, hash, signature)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#Account-OwnerAdded[`++OwnerAdded(new_owner_guid)++`]
+* xref:#Account-OwnerRemoved[`++OwnerRemoved(removed_owner_guid)++`]
+--
+
+[.contract-item]
+[[Account-__validate_deploy__]]
+==== `[.contract-item-name]#++__validate_deploy__++#++(self: @ContractState, class_hash: felt252, contract_address_salt: felt252, _public_key: felt252) → felt252++` [.item-kind]#external#
+
+Validates a DeployAccount transaction.
+
+Returns the short string `'VALID'` if valid, or any other value otherwise.
+
+[.contract-item]
+[[Account-__execute__]]
+==== `[.contract-item-name]#++__execute__++#++(ref self: ContractState, calls: Array<Call>) → Array<Span<felt252>>++` [.item-kind]#external#
+
+See xref:ISRC6-\\__execute__[ISRC6::\\__execute__].
+
+[.contract-item]
+[[Account-__validate__]]
+==== `[.contract-item-name]#++__validate__++#++(self: @ContractState, calls: Array<Call>) → felt252++` [.item-kind]#external#
+
+See xref:ISRC6-\\__validate__[ISRC6::\\__validate__].
+
+[.contract-item]
+[[Account-is_valid_signature]]
+==== `[.contract-item-name]#++is_valid_signature++#++(self: @ContractState, hash: felt252, signature: Array<felt252>) → felt252++` [.item-kind]#external#
+
+See xref:ISRC6-is_valid_signature[ISRC6::is_valid_signature].
+
+[.contract-item]
+[[Account-supports_interface]]
+==== `[.contract-item-name]#++supports_interface++#++(self: @ContractState, interface_id: felt252) → bool++` [.item-kind]#external#
+
+Returns whether a contract implements a given interface or not.
```

Comment:
```suggestion
Returns whether a contract implements a given interface or not.
```
please add a link to SRC5 / introspection section

Reply:
I'm planning to in the introspection PR.